File: python/commands/ping.py
import discord
from typing import List

def attach(client: discord.Client, message: discord.Message):
    return Ping(client, message)

class Ping:
    def __init__(self, client: discord.Client, message: discord.Message):
        self.client = client
        self.message = message

    async def run(self, args: List[str]) -> None:
        client: discord.Client = self.client
        message: discord.Message = self.message
        await message.reply("Pong!")

File: swift/commands/ping.swift
import DiscordBM

struct Ping: Command {
    let discord: any DiscordClient
    let message: Gateway.MessageCreate

    init(discord: any DiscordClient, message: Gateway.MessageCreate) {
        self.discord = discord
        self.message = message
    }

    func run(args: [String]) async throws {
        let _ = try await discord.createMessage(
            channelId: message.channel_id, 
            payload: .init(content: "Pong", message_reference: .init(message_id: message.id))
        )
    }
}